tests/input/semdata.yaml

```diff
@@ -0,0 +1,80 @@
+---
+# This extends the list of prefixes that are already defined in the context
+prefixes:
+  sem: https://w3id.com/emmo/domain/sem/0.1#
+  semdata: https://he-matchmaker.eu/data/sem/
+  sample: https://he-matchmaker.eu/sample/
+  mat: https://he-matchmaker.eu/material/
+  dm: http://onto-ns.com/meta/characterisation/0.1/SEMImage#
+  parser: http://sintef.no/dlite/parser#
+  gen: http://sintef.no/dlite/generator#
+
+
+# List of documented datasets
+datasets:
+  - "@id": semdata:SEM_cement_batch2/77600-23-001/77600-23-001_5kV_400x_m001
+    "@type": sem:SEMImage
+    title: SEM image of cement
+    description: Back-scattered SEM image of cement sample 77600 from Heidelberg, polished with 1 µm diamond compound.
+    creator: Sigurd Wenner
+    contactPoint: Sigurd Wenner <Sigurd.Wenner@sintef.no>
+    inSeries: semdata:SEM_cement_batch2/77600-23-001
+
+    datamodel: http://onto-ns.com/meta/characterisation/0.1/SEMImage
+    datamodelStorage: https://github.com/HEU-MatCHMaker/DataDocumentation/blob/master/SEM/datamodels/SEMImage.yaml
+    #mappingURL: https://raw.githubusercontent.com/HEU-MatCHMaker/DataDocumentation/refs/heads/master/SEM/datamodels/SEMImage.ttl
+
+    # Contextual documentation of the dataset
+    statements:
+      - ["@id", "sem:fromSample", "sample:SEM_cement_batch2/77600-23-001"]
+      - ["@id", "emmo:isDescriptionOf", "mat:concrete1"]
+
+    distribution:
+      #downloadURL: sftp://nas.aimen.es/P_MATCHMAKER_SHARE_SINTEF/SEM_cement_batch2/77600-23-001/77600-23-001_5kV_400x_m001.tif
+      downloadURL: https://github.com/EMMC-ASBL/tripper/raw/refs/heads/dataset/tests/input/77600-23-001_5kV_400x_m001.tif
+      mediaType: image/tiff
+      parser: parser:sem_hitachi
+      generator: gen:sem_hitachi
```

Comment:
OK, after a discussion we have agreed to continue with datamodel at the dataset-level. If a different datamodel is used to describe the same raw data it will be a different dataset. However, I still think we should _NOT_ have a generator when describig this particular dataset as it is not a datasink. We should have a separate description of a possible datasink. A desitribution description of a datasink can have both parser and a generator. The generator is necessary. The parser optional, and is intended to be used to document the generated dataset as a dataset after it has been generated.